janusgraph-hadoop/src/main/java/org/janusgraph/hadoop/MapReduceIndexManagement.java

```diff
@@ -67,17 +68,22 @@ public MapReduceIndexManagement(JanusGraph g) {
         this.graph = (StandardJanusGraph)g;
     }
 
+    public JanusGraphManagement.IndexJobFuture updateIndex(Index index, SchemaAction updateAction) throws BackendException {
+        return updateIndex(index, updateAction, new Configuration());
+    }
+
     /**
      * Updates the provided index according to the given {@link SchemaAction}.
      * Only {@link SchemaAction#REINDEX} and {@link SchemaAction#REMOVE_INDEX} are supported.
      *
      * @param index the index to process
      * @param updateAction either {@code REINDEX} or {@code REMOVE_INDEX}
+     * @param hadoopConf
      * @return a future that returns immediately;
      *         this method blocks until the Hadoop MapReduce job completes
      */
     // TODO make this future actually async and update javadoc @return accordingly
-    public JanusGraphManagement.IndexJobFuture updateIndex(Index index, SchemaAction updateAction)
+    public JanusGraphManagement.IndexJobFuture updateIndex(Index index, SchemaAction updateAction, Configuration hadoopConf)
```

Comment:
I prefer to ignore the codacy warning, i.e.

> The method 'updateIndex(Index, SchemaAction, Configuration)' has an NPath complexity of 336, current threshold is 200

to keep this PR as simple as possible, just in case refactoring changes existing functionality accidentally.